packages/js/client/src/wasm/WasmWeb3Api.ts

```diff
@@ -56,7 +56,9 @@ export class WasmWeb3Api extends Api {
     // TODO: come up with a better future-proof solution
     while (threadsActive >= maxThreads) {
       // Wait for another thread to become available
-      await new Promise((resolve) => setTimeout(() => resolve(), 500));
+      await new Promise((resolve: (value?: unknown) => void) =>
+        setTimeout(() => resolve(), 500)
+      );
```

Comment:
you are using this in couple of places. Could be extracted to "sleep(ms: number)" utility method

Reply:
Will do :D